dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/security/DefaultSecurityService.java

```diff
@@ -242,7 +243,7 @@ public void prepareUserForInvite( User user )
 
         if ( user.getUsername() == null || user.getUsername().isEmpty() )
         {
-            String username = "invite-" + user.getEmail() + "-" + CodeGenerator.generateUid();
+            String username = "invite-" + user.getEmail() + "-" + CodeGenerator.generateUid().toLowerCase();
```

Comment:
hm.. isn't the email good enough? and should the email not be unique you cannot invite that user as it obviously already is a user? - a user with this generated name is kind of annoying